src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int RANDNUMBER = 101;
```

Comment:
Where is access modifier of this class-member? Let's better choose name for this constant. For example `UPPER_BOUND` or `MAX_NUMBER`. Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.